### PR TITLE
Add default consumer terms and privacy policy

### PR DESCRIPTION
**Pull request by cassiewallace**
[Clubhouse #6205](https://app.clubhouse.io/flow/story/6205/create-default-privacy-policy-and-terms)

Add a default version of the consumer terms and privacy policy using the term "the merchant". This will be used when we are in development (as this is required content in order to set up Contentful) and do not yet have the finalized terms.

### CODE REVIEW

File: util/content/default/privacy-policy.html [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+        Policy, we will post the updated policy at
+        <a href="https://www.flow.io/help/privacy">https://www.flow.io/help/privacy</a>.
+        We reserve the right to modify this Privacy Policy at any time, so please
+        come to this link to review it frequently. If we make significantly material
+        changes to this Privacy Policy, we will notify you by means of a notice on
+        <a href="http://www.flow.io">www.flow.io</a> prior
+        to the change becoming effective.
+      </p>
+
+      <h3><a name=s12></a>What are the Terms &amp; Conditions and
```
Comment (nathanstilwell) [inline util/content/default/privacy-policy.html]: What does this `name=s12` do?

Comment (cassiewallace) [reply util/content/default/privacy-policy.html]: That's a wonderful question. It's the same in the original--I'll investigate tomorrow.

Comment (cassiewallace) [reply util/content/default/privacy-policy.html]: In the meantime I'll merge; if we want to remove this I'll do it across all of them.

**Comment by cassiewallace** [comment]
Need @nathanstilwell to do `aws sync` now :)

**Comment by mbryzek** [comment]
@cassiewallace synced

**Pull request merged**